synapse/tests/test_lib_stormlib_cell.py

```diff
@@ -1,9 +1,37 @@
 import synapse.exc as s_exc
 
-import synapse.lib.stormlib.cell as s_cell
+import synapse.lib.cell as s_cell
+import synapse.lib.const as s_const
+import synapse.lib.stormsvc as s_stormsvc
+import synapse.lib.stormlib.cell as s_stormlib_cell
 
 import synapse.tests.utils as s_test
 
+class TestsvcApi(s_cell.CellApi, s_stormsvc.StormSvc):
```

Comment:
for reference we don't actually need the target cell to implement the StormSvc convention.

Reply:
we could also reuse one of the services from `test_lib_stormsvc` too.